cmd/dump.go

```diff
@@ -85,7 +90,7 @@ func runDump(ctx *cli.Context) error {
 
 	dbDump := path.Join(tmpWorkDir, "gitea-db.sql")
 
-	fileName := fmt.Sprintf("gitea-dump-%d.zip", time.Now().Unix())
+	fileName := fmt.Sprintf(ctx.String("file"), time.Now().Unix())
```

Comment:
Also if the value of `--file` is provided by the user and doesn't have a `%d` format directive, wouldn't that ruin the value of `fileName`. If I remember correctly, when a string doesn't have a format directive, the resulting output is something like `%!(EXTRA int64=1257894000)`.  So in this case, `fileName` will end up being `userSuppliedFileName%!(EXTRA int64=1257894000)` . Or am I thinking this too far ?

Reply:
you, sir, are completely right. updated the logic.